ts/features/fims/history/components/__tests__/FimsHistoryListItemPicker.test.tsx

```diff
@@ -0,0 +1,109 @@
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { createStore } from "redux";
+import { ServicePublic } from "../../../../../../definitions/backend/ServicePublic";
+import { Access } from "../../../../../../definitions/fims_history/Access";
+import { applicationChangeState } from "../../../../../store/actions/application";
+import { appReducer } from "../../../../../store/reducers";
+import { renderScreenWithNavigationStoreContext } from "../../../../../utils/testWrapper";
+import * as FETCH_HOOKS from "../../../common/hooks";
+import { FimsHistoryListItemPicker } from "../FimsHistoryListItemPicker";
+import * as LIST_ITEMS from "../FimsHistoryListItems";
+
+const mockServicePublic = { organization_name: "TEST" } as ServicePublic;
+const mockAccess: Access = {
+  id: "TESTING",
+  redirect: { display_name: "TESTING", uri: "TESTING" },
+  service_id: "TESTING_SID",
+  timestamp: new Date(0)
+};
+
+describe("FimsHistoryListItem", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("should match snapshot for a some state", () => {
```

Comment:
What about defining mocks for `FimsHistorySuccessListItem`, `FimsHistoryFailureListItem` (like `ts/features/messages/components/Home/__mocks__/PreconditionsContent.tsx`, as an example) and create snapshots for each state of the input pot?